package.json

```diff
@@ -82,7 +82,7 @@
     "build": "gatsby build --prefix-paths --log-pages",
     "dev": "gatsby develop",
     "serve": "npm run build && gatsby serve --prefix-paths -o",
-    "format": "prettier \"**/*.{ts,tsx}\" \"!out\" --write --check",
+    "format": "prettier . --write --check",
```

Comment:
Dangerous change maybe?

Reply:
It's suggested to use like that in docs https://prettier.io/docs/en/install.html